main/lsp/LSPTypechecker.h

```diff
@@ -207,5 +180,15 @@ class LSPTypecheckerDelegate {
     const core::GlobalState &state() const;
 };
 
+class AutocorrectFlusher : public core::ErrorFlusher {
```

Comment:
Oh, what I was saying before is that you don't have to mention `AutocorrectFlusher` in a `.h` file at all. You can just define the entire class in the `.cc` file!

e.g. this is totally fine in a .cc file:

```c++
class AutocorrectFlusher : public core::ErrorFlusher {
  public:
    std::vector<std::unique_ptr<core::Error>> collectedErrors;
    AutocorrectFlusher() = default;
    ~AutocorrectFlusher() = default;
    void flushErrors(spdlog::logger &logger, std::vector<std::unique_ptr<core::ErrorQueueMessage>> errors,
                     const core::GlobalState &gs, core::FileRef file) override {
      // implementation here!
    }
};
```

This way, this class doesn't clutter up the public interface to `LSPTypechecker` (the header file).